Corona-Warn-App/src/main/res/layout/include_submission_test_result.xml

```diff
@@ -57,6 +57,17 @@
                 app:layout_constraintStart_toStartOf="@+id/guideline_start"
                 app:layout_constraintTop_toBottomOf="@+id/submission_test_result_subtitle" />
 
+            <de.rki.coronawarnapp.ui.submission.consentstatus.ConsentStatusView
+                android:id="@+id/consent_status"
+                android:layout_width="@dimen/match_constraint"
+                android:layout_height="wrap_content"
+                android:layout_marginBottom="@dimen/spacing_medium"
+                android:focusable="true"
+                android:visibility="@{FormatterSubmissionHelper.formatTestResultPendingStepsVisible(uiState.deviceUiState)}"
```

Comment:
We should refactor the FormatterSubmissionHelper at some point.
Not a blocker for this PR, just a note.

Reply:
For sure we should place it into refactoring list.